app/src/app.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+	state = {
+			isLoading: false,
+			groups: []
+	}
+	
+  async componenetDidMount(){
+		const response = await fetch('/api/groups');
```

Comment:
the task was not to make fetch call rather a mock response.

you could have done 
```
const body = [{id:1, name:"Denver JUG"}, {id: 2, name : "Utah JUG"}]

```